KarhooUISDK/Screens/CheckoutScreen/Checkout/CheckoutView.swift

```diff
@@ -83,36 +82,10 @@ struct CheckoutView: View {
                         if viewModel.legalNoticeViewModel.shouldShowView {
                             HStack {
                                 Spacer()
-                                Button(
-                                    action: {
-                                        withAnimation {
-                                            showLegalNotice.toggle()
-                                            if showLegalNotice {
-                                                scrollViewProxy.scrollTo(Constants.legalNoticeViewId, anchor: .bottom)
-                                            }
-                                        }
-                                    },
-                                    label: {
-                                        HStack(spacing: UIConstants.Spacing.small) {
-                                            Text(UITexts.Booking.legalNotice)
-                                                .foregroundColor(Color(KarhooUI.colors.accent))
-                                                .font(Font(KarhooUI.fonts.captionSemibold()))
-                                            Image("kh_uisdk_drop_down_icon", bundle: .current)
```

Comment:
Isn't the icon used anywhere else in the UISDK? The asset has been deleted so just making sure it isn't affecting other parts of the UI SDK